index.js

```diff
@@ -99,7 +99,7 @@ module.exports = function(options) {
       // The request is outside the scope of this specification.
       if (!ctx.get('Access-Control-Request-Method')) {
         // this not preflight request, ignore it
-        return next();
+        return;
```

Comment:
@kurayama I think this is debatable and if the spec says it's of scope, then it is my understanding we should let the request pass on to the next middleware.